src/inverse.jl

```diff
@@ -0,0 +1,12 @@
+for d in (:norm,)
+    @eval begin
+        InverseFunctions.inverse(::typeof($(Symbol(d,:cdf)))) = $(Symbol(d, :invcdf))
+        InverseFunctions.inverse(::typeof($(Symbol(d,:ccdf)))) = $(Symbol(d, :invccdf))
+        InverseFunctions.inverse(::typeof($(Symbol(d,:logcdf)))) = $(Symbol(d, :invlogcdf))
+        InverseFunctions.inverse(::typeof($(Symbol(d,:logccdf)))) = $(Symbol(d, :invlogccdf))
+        InverseFunctions.inverse(::typeof($(Symbol(d,:invcdf)))) = $(Symbol(d, :cdf))
+        InverseFunctions.inverse(::typeof($(Symbol(d,:invccdf)))) = $(Symbol(d, :ccdf))
+        InverseFunctions.inverse(::typeof($(Symbol(d,:invlogcdf)))) = $(Symbol(d, :logcdf))
+        InverseFunctions.inverse(::typeof($(Symbol(d,:invlogccdf)))) = $(Symbol(d, :logccdf))
+    end
+end
```

Comment:
I guess it would be a bit simpler to not use `@eval` here:
```suggestion
InverseFunctions.inverse(::typeof(norncdf)) = norminvcdf
InverseFunctions.inverse(::typeof(norminvcdf)) = normcdf

InverseFunctions.inverse(::typeof(normccdf)) = norminvccdf
InverseFunctions.inverse(::typeof(norminvccdf)) = normccdf

InverseFunctions.inverse(::typeof(normlogcdf)) = norminvlogcdf
InverseFunctions.inverse(::typeof(norminvlogcdf)) = normlogcdf

InverseFunctions.inverse(::typeof(normlogccdf)) = norminvlogccdf
InverseFunctions.inverse(::typeof(norminvlogccdf)) = normlogccdf
```

Reply:
I wanted to keep it flexible, in case there's some other family of functions with only one argument in the future. Well, probably not likely.